troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cannot Load iframe on iOS devices when using Cordova 5.0.0 or later
+description: Cannot load iframe on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova 5.0.0 or later apps do not load iframe on iOS
```

Comment:
This would appear as a title in the search results. Apply title style capitalization.